external_worker/cloud/cloud-init.txt

```diff
@@ -0,0 +1,34 @@
+#cloud-config
+package_upgrade: true
+packages:
+  - docker
+  - docker-compose
+  - nodejs
+  - npm
+write_files:
+  - owner: enigma:enigma
+    path: /home/enigma/docker-compose.yml
+    content: |
+      version: "2"
+      services:
+        worker:
+          hostname: worker
+          image: enigmampc/worker_sw:0.0.1
```

Comment:
As mentioned below, I would default to Hardware mode, since this are files meant for deployment on a real network: `worker_hw`